documentation/docs/basics/authentication/example_is_sdk.md

```diff
@@ -0,0 +1,90 @@
+---
+image: /img/integration-services/logo/integration_services.png
+description: This section will show you an example on how to authenticate using the IOTA IS-SDK npm package.
+keywords:
+- authentication
+- jwt
+- nonce
+- not-authenticated
+- 401
+- nodejs
+- is-sdk
+- npm
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Integration Services SDK
+
+In this example you will learn how to authenticate your identity using our own **IOTA IS-SDK** npm package. This will be the easiest way to authenticate your identity since we are handling decoding, hashing and signing directly in the package itself. Make sure to read the [general authentication concept](https://wiki.iota.org/integration-services/authentication) for better understanding on how we authenticate users.
```

Comment:
```suggestion
In this example, you will learn how to authenticate your identity the [**IOTA IS-SDK** npm package](https://www.npmjs.com/package/iota-is-sdk). Since the package will handle decoding, hashing, and signing, this is the simplest way to authenticate your identity. Make sure to read the [general authentication concept](https://wiki.iota.org/integration-services/authentication), so you can fully understand the [Authentication Workflow](https://wiki.iota.org/integration-services/authentication#authentication-workflow).
```

Reply:
The meaning of the first sentence is not what I intended to say anymore. The identity is not the package itself. The package is only used for authenticating the identity. I will commit your suggestions but will change the first sentence a little, if thats fine with you.

Reply:
```suggestion
In this example, you will learn how to authenticate your identity using the [**IOTA IS-SDK** npm package](https://www.npmjs.com/package/iota-is-sdk). Since the package will handle decoding, hashing, and signing, this is the simplest way to authenticate your identity. Make sure to read the [general authentication concept](https://wiki.iota.org/integration-services/authentication), so you can fully understand the [Authentication Workflow](https://wiki.iota.org/integration-services/authentication#authentication-workflow).
```

Reply:
@lucas-tortora what do you think?